infra/recipes/terraform/azure/environments/sandbox/main.tf

```diff
@@ -1,35 +1,82 @@
-resource "azurerm_resource_group" "openhouse_sandbox" {
+data "azurerm_resource_group" "openhouse_sandbox" {
     name = var.resource_group_name
-    location = var.resource_group_location
+}
+
+resource "random_string" "storage_name" {
+    length = 5
+    special = false
+    lower = true
+    upper = false
+}
+
+locals {
+    storage_account_name = "openhousestorage${random_string.storage_name.result}" // added random string of numbers to make it unique
+    container_name = "blobcontainer"
+    db_username = "azureadmin"
+    db_password = "Pa33word"
+    db_name = "openhouse-sandbox-db"
+    db_server_name = "openhouse-sandbox-mysql-server"
 }
 
 module "vm" {
     source = "../../modules/vm"
+    depends_on = [ data.azurerm_resource_group.openhouse_sandbox ]
     virtual_network_name = "openhouse-sandbox-network"
     resource_group_name = var.resource_group_name
     subnet_name = "openhouse-sandbox-subnet"
 }
 
 module "mysql" {
     source = "../../modules/mysql"
+    depends_on = [ module.vm ]
     subnet_id = module.vm.subnet_id
-    resource_group_name = azurerm_resource_group.openhouse_sandbox.name
-    server_name = "openhouse-sandbox-mysql-server"
-    db_admin_login = "azureadmin"
-    db_admin_password = "Pa33word"
-    db_name = "openhouse-sandbox-db"
+    resource_group_name = data.azurerm_resource_group.openhouse_sandbox.name
+    server_name = local.db_server_name
+    db_admin_login = local.db_username
+    db_admin_password = local.db_password
+    db_name = local.db_name
+    server_sku = "B_Standard_B2ms"
```

Comment:
Can you comment here what is the cpu core and memory config for the model?

Reply:
Yes